alphadia/libtransform.py

```diff
@@ -279,6 +280,10 @@ def __init__(
         peptdeep_model_path : str, optional
             Path to a folder containing PeptDeep models. If not provided, the default models will be used.
 
+        peptdeep_model_type : str, optional
+            Use other peptdeep models provided by the peptdeep model manager. Default is None.
```

Comment:
`Default is None, which means the peptdeep default model ("generic") is being used.